keydata.go

```diff
@@ -42,7 +42,7 @@ import (
 )
 
 const (
-	KeyDataVersion int = 2
```

Comment:
It's probably cleaner to just define `KeyDataVersion` as a variable here rather than having a mockable function, and then `MockKeyDataVersion` can just set this instead.